app/models/owner_statistic.rb

```diff
@@ -40,50 +40,50 @@ def collection_ids
+  def mention_count(days=nil)
+    PageArticleLink.where(article_id: owner_subjects.ids).where(date_range_clause(days, 'created_on')).count
   end
 
-  def contributor_count(last_days=nil)
-    User.joins(:deeds).where(deeds: {collection_id: collection_ids}).where("#{last_days_clause(last_days, 'deeds.created_at')}").distinct.count
+  def contributor_count(days=nil)
+    User.joins(:deeds).where(deeds: {collection_id: collection_ids}).where(date_range_clause(days, 'deeds.created_at')).distinct.count
   end
 
-  def comment_count(last_days=nil)
-    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::NOTE_ADDED).where("#{last_days_clause(last_days)}").count
+  def comment_count(days=nil)
+    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::NOTE_ADDED).where(date_range_clause(days)).count
 
   end
 
-  def transcription_count(last_days=nil)
-    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_TRANSCRIPTION).where("#{last_days_clause(last_days)}").count
+  def transcription_count(days=nil)
+    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_TRANSCRIPTION).where(date_range_clause(days)).count
   end
 
-  def edit_count(last_days=nil)
-    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_EDIT).where("#{last_days_clause(last_days)}").count
+  def edit_count(days=nil)
+    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_EDIT).where(date_range_clause(days)).count
   end
 
-  def index_count(last_days=nil)
-    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_INDEXED).where("#{last_days_clause(last_days)}").count
+  def index_count(days=nil)
+    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_INDEXED).where(date_range_clause(days)).count
   end
 
-  def translation_count(last_days=nil)
-    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_TRANSLATED).where("#{last_days_clause(last_days)}").count
+  def translation_count(days=nil)
+    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::PAGE_TRANSLATED).where(date_range_clause(days)).count
   end
 
-  def ocr_count(last_days=nil)
-    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::OCR_CORRECTED).where("#{last_days_clause(last_days)}").count
+  def ocr_count(days=nil)
+    Deed.where(collection_id: collection_ids).where(deed_type: DeedType::OCR_CORRECTED).where(date_range_clause(days)).count
   end
 
-  def last_days_clause(last_days, column = "created_at")
-    clause = ""
-    if last_days
-      timeframe = last_days.days.ago
-      clause = "#{column} >= '#{timeframe}'"
+  def date_range_clause(days, column = "created_at")
+    if days.nil?
+      return ""
+    elsif days.is_a? Integer
+      days = (days.days.ago)..(Time.now)
     end
-    return clause
+    return {column.to_sym => days}
```

Comment:
That's fascinating -- I had no idea that active record where clauses could take a Ruby range to create a SQL `BETWEEN ... AND ...`!